admin/test/test_release.py

```diff
@@ -903,6 +903,8 @@ def test_fake_rpm(self):
                          '<oldhash>-metadata.xml'):
                 'metadata for: existing_package.rpm',
         }
+        # Copy before passing to FakeAWS
+        expected_keys = existing_s3_keys.copy()
```

Comment:
Change `FakeAWS` so that this isn't necessary.  ie, `FakeAWS` should be doing the copy instead of mutating the argument passed to it.